packages/react/src/formElements/MdAutocompleteAsync.tsx

```diff
@@ -0,0 +1,236 @@
+                    id={autocompleteId}
+                    aria-describedby={helpText && helpText !== '' ? `md-autocomplete_help-text_${autocompleteId}` : undefined}
+                    className={inputClassNames}
+                    value={open ? autocompleteValue : displayValue}
+                    type='text'
+                    tabIndex={0}
+                    onFocus={() => !disabled && onFocus()}
+                    onChange={(e) => setAutocompleteValue(e.target.value)}
+                    placeholder={placeholder}
+                    disabled={disabled}
+                    autoComplete={autoComplete}
+                    {...otherProps}
+                >
+                </input>
+                <div aria-hidden="true" className="md-autocomplete__input-icon">
+                    <MdChevronIcon transform={`rotate(${open ? '180' : '0'})`} />
+                </div>
+
+                {results && results.length > 0 &&
+                    <div
+                        aria-labelledby={label && label !== '' ? `md-autocomplete_label_${autocompleteId}` : undefined}
+                        role="listbox"
+                        id={`md-autocomplete__dropdown_${autocompleteId}`}
+                        className="md-autocomplete__dropdown"
+                    >
+                        {results.map(option => (
+                            <button
+                                role="option"
+                                key={`md-autocomplete-option-${autocompleteId}-${option.value}`}
+                                id={`md-autocomplete-option-${autocompleteId}-${option.value}`}
+                                type='button'
+                                tabIndex={open ? 0 : -1}
+                                className={optionClass(option)}
+                                onClick={() => open && handleOptionClick(option)}
+                            >
+                                <div className="md-autocomplete__dropdown-item-text">{displayValueAndText ? `${option.value} ${option.text}` : option.text}</div>
+                                {isSelectedOption(option) &&
+                                    <div className="md-autocomplete__dropdown-item-clear" title="Klikk for å fjerne valg">
+                                        <MdXIcon />
+                                    </div>
+                                }
+                            </button>
+                        ))}
+                    </div>
+                }
+            </MdClickOutsideWrapper>
+
+            {error && errorText && errorText !== '' &&
+                <div className="md-autocomplete__error">{errorText}</div>
+            }
+        </div>
+    )
+}
+
+export function RequiredAsterisk(): ReactElement {
+    return (
+        <span className="md-autocomplete__asteriks" aria-label='obligatorisk'> &#42;</span>
+    );
+}
+
+export default MdAutocompleteAsync;
```

Comment:
Missing newline at end of file. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline